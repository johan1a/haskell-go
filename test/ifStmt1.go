func main() {
if b { a }
}
