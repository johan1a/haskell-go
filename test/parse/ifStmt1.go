package main
import "fmt";
func main() {
if b {  
}
}
