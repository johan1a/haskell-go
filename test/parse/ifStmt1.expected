SourceFile (Package "main") [TopLevelDecl2 (FunctionDecl2 "main" (Signature1 []) (Block [IfStmt (Ifstmt1 (UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand2 (OperandName1 "b"))))) (Block []))]))]
